.github/workflows/DeployStaticExecutables.yaml

```diff
@@ -0,0 +1,106 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+name: Deploy static executables
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        type: string
+        required: true
+        description: The name of the release tag
+
+jobs:
+  deploy_static_execs:
+    name: Deploy static executables and libraries
+    runs-on: ubuntu-latest
+    # Running in a protected environment that provides the necessary secrets
+    environment: release
```

Comment:
Also use `deploy-containers` environment? At least that's what DemoContainers.yaml does.

Reply:
The deploy doesn't have access to the release token.

Reply:
You don't need the release token I think

Reply:
I don't think we need to get prompted before upload? There is a check that the CCE run is consistent with previous results, so the action should fail if something went wrong. I guess then I'll just not specify an env. I'll do this and push. If I misunderstood just let me know and I'll fix it :)